arkouda/strings.py

```diff
@@ -8,9 +8,6 @@
 import numpy as np
 import json
 
-global verbose
-global pdarrayIterThresh
```

Comment:
there is a reference to this in the Strings class. lone 105 or so please fix

Reply:
fixed